### PR TITLE
Couple corrections and API extensions

### PR DESCRIPTION
**Pull request by digetx**

### CODE REVIEW

File: tegra/tegra.c [hunk 1]
```diff
@@ -121,6 +120,7 @@ int drm_tegra_bo_new(struct drm_tegra_bo **bop, struct drm_tegra *drm,
 {
 	struct drm_tegra_gem_create args;
 	struct drm_tegra_bo *bo;
+	bool bo_cleaned = false;
```
Comment (kusma) [inline tegra/tegra.c]: This is unused in this commit...

Comment (digetx) [inline tegra/tegra.c]: Good catch 👍

Comment (digetx) [inline tegra/tegra.c]: Fixed ;)

**Comment by kusma** [comment]
Looks good!

**Pull request merged**